ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -122,23 +133,32 @@ def __init__(
             )
         self._loaded = True
 
-        rl_init_parameters_in = UnityRLInitializationInputProto(seed=seed)
+        rl_init_parameters_in = UnityRLInitializationInputProto(
+            seed=seed,
+            communication_version=self.API_VERSION,
+            package_version=mlagents_envs.__version__,
+        )
         try:
             aca_output = self.send_academy_parameters(rl_init_parameters_in)
             aca_params = aca_output.rl_initialization_output
         except UnityTimeOutException:
             self._close()
             raise
         # TODO : think of a better way to expose the academyParameters
```

Comment:
Not exactly related to this change, but should this be removed?

Reply:
Removed.